packages/bio_io/bio_io.0.7.0/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "A library for reading and writing common file formats used in bioinformatics like FASTA files"
+maintainer: ["Ryan M. Moore"]
+authors: ["Ryan M. Moore"]
+homepage: "https://github.com/mooreryan/bio_io"
+doc: "https://mooreryan.github.io/bio_io/"
+bug-reports: "https://github.com/mooreryan/bio_io/issues"
+depends: [
+  "dune" {>= "3.2"}
+  "base" {>= "v0.16" & < "v0.17"}
```

Comment:
```suggestion
  "base" {>= "v0.16" & < "v0.17"}
```
Are you sure we need these upper bounds? 
I suggest we remove them and then we add them manually if a future release breaks the package

Reply:
No I'm not sure we need them.  (I started putting them in for my local packages as the Jane Street minor version increments generally break stuff.)

I will remove the upper bounds.